sections/sprint-1/networking.tex

```diff
@@ -263,3 +263,62 @@ \subsection{IPv4 and IPv6}\label{ipv4-ipv6}
 IPv6 addresses are usually written as eight 16-bit hexadecimal numbers \cite{socketnetworking}.
 The higher-order bits of the 128-address imply the type.
 For this project, this should not have a significant impact, and either can be used.
+
+
+\subsection{Format of data}
+As it is needed to send data over the network for our game, it is required to define the format it is sent as.
+The network format for the field anchor, player position and goal position is \texttt{0xyyyyxxxxiisstt}, where \texttt{t} is the type, \texttt{s} is the timestamp, \texttt{i} is the ID, \texttt{xxxx} is the \texttt{x} position and \texttt{yyyy} is the \texttt{y} position.
+These formats can be seen on \autoref{Tab:anchor-format}, \autoref{Tab:player-format} and \autoref{Tab:player-format}.
+These are almost identical, as the same information is required.
+The only difference between them is the type, and the context of the IDs.
+The type is at the end of the hexadecimal, so that it is easy to bitshift, and then switch on which type it is.
+After the type the timestamp is placed, which is used to detect outdated data.
+\\\\
+It was decided to use 2 and 4 hexadecimal, so that it is in byte format and hereby straightforward to work with in \texttt{C\#}.
+Given the highest number that can be in two bytes are $2^{16} - 1$, the maximum number for the \texttt{x} and \texttt{y} positions are 65.535. 
+\begin{table}[h!]
+    \centering
+    \begin{tabular}{|l|l|l|l|l|}
+    \hline
+    \begin{tabular}[c]{@{}l@{}}POSY (y)\\ (0-65.535)\end{tabular} & \begin{tabular}[c]{@{}l@{}}POSX (x)\\ (0-65.535)\end{tabular} & \begin{tabular}[c]{@{}l@{}}ID (i)\\ (0-3)\end{tabular} & \begin{tabular}[c]{@{}l@{}}TIMESTAMP (s)\\ (0-255)\end{tabular} & \begin{tabular}[c]{@{}l@{}}TYPE (t)\\ (0)\end{tabular} \\ \hline
+    4 hexadecimal                                                 & 4 hexadecimal                                                 & 2 hexadecimal                                          & 2 hexadecimal                                                 & 2 hexadecimal                                        \\ \hline
```

Comment:
Bør skrives som bits, da det er standarden for det

![image](https://user-images.githubusercontent.com/21969475/77646190-57e10980-6f64-11ea-9e00-16089684cf5f.png)

Reply:
bare som lidt ekstra info så svarer 1 hexadecimal til 4 bits.